### DISCUSSION TITLE
Are there any unique characteristics about an individual player

### DISCUSSION THREAD

**Discussion by noahantoun** [cwendt94/espn-api#271] [ESPN API]
I am trying to connect this nfl league data. The difficulty is finding a connection point. There is the espnff id, but that is only espnff specific. A common one used in other sources is GSIS, I don't know if that is available. Name isn't unique enough, positions and teams change. Is there any unique info on a player that can be used along with name to link to another stats DB? I was thinking birthdate.

**Comment by cwendt94** [comment]
I'm not sure if ESPN provides any unique id outside of their internal one. Also I have checked some of the data and have not seen birthdate as well.
